lib/fluent/plugin/out_sumologic.rb

```diff
@@ -397,18 +404,49 @@ def write(chunk)
         fields = [fields,@custom_fields].compact.join(",")
       end
 
-      @log.debug { "Sending #{messages.count} #{@data_type} records with source category '#{source_category}', source host '#{source_host}', source name '#{source_name}'." }
-
-      @sumo_conn.publish(
-          messages.join("\n"),
-          source_host         =source_host,
-          source_category     =source_category,
-          source_name         =source_name,
-          data_type           =@data_type,
-          metric_data_format  =@metric_data_format,
-          collected_fields    =fields,
-          dimensions          =@custom_dimensions
-      )
+      retries = 0
+      start_time = Time.now
+      sleep_time = @retry_min_interval
+
+      while true
+        begin
+          @log.debug { "Sending #{messages.count} #{@data_type} records with source category '#{source_category}', source host '#{source_host}', source name '#{source_name}', chunk  #{chunk_id}." }
```

Comment:
Would it be worth adding the value of `retries` into this debug log? What do you think?

Reply:
It'd be good to have that information in the warning and info logs below as well.